src/utils/createI13nNode.js

```diff
@@ -11,6 +11,7 @@ var hoistNonReactStatics = require('hoist-non-react-statics');
 var PROPS_TO_FILTER = [
     'bindClickEvent',
     'follow',
+    'followLink',
```

Comment:
Instead of leaking `followLink` prop from fluxible-router NavLink into this, we should plan for removing the code referring to `props.followLink`.   Print warning for now, and remove the prop later.